spec/factories/payments.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :payment do
+    # Payment Factory
```

Comment:
Are you planning to improve the payment model later?

Reply:
No apu, in our ER Diagram, we just had the `Payment` model as a dummy model. We'll just make sure that every `Ticket` has a `Payment`. Nothing much.